.github/workflows/prod.yml

```diff
@@ -13,60 +16,24 @@ jobs:
       matrix:
         target:
           [
-            { os: ubuntu, version: 20.04, dist: focal, package-type: DEB },
-            { os: ubuntu, version: 22.04, dist: jammy, package-type: DEB },
-            { os: debian, version: 10, dist: buster, package-type: DEB },
-            { os: debian, version: 11, dist: bullseye, package-type: DEB },
+            { os: debian, version: 10, arch: amd64, dist: buster, package-type: DEB, repo: microsoft-debian-buster-prod-apt },
+            { os: debian, version: 10, arch: arm64, dist: buster, package-type: DEB, repo: microsoft-debian-buster-prod-apt },
+            { os: debian, version: 11, arch: amd64, dist: bullseye, package-type: DEB, repo: microsoft-debian-bullseye-prod-apt },
+            { os: debian, version: 11, arch: arm64, dist: bullseye, package-type: DEB, repo: microsoft-debian-bullseye-prod-apt },
+            { os: debian, version: 12, arch: amd64, dist: bookworm, package-type: DEB, repo: microsoft-debian-bookworm-prod-apt },
+            { os: debian, version: 12, arch: arm64, dist: bookworm, package-type: DEB, repo: microsoft-debian-bookworm-prod-apt },
+            { os: ubuntu, version: 20.04, arch: amd64, dist: focal, package-type: DEB, repo: microsoft-ubuntu-focal-prod-apt },
+            { os: ubuntu, version: 20.04, arch: arm64, dist: focal, package-type: DEB, repo: microsoft-ubuntu-focal-prod-apt },
+            { os: ubuntu, version: 22.04, arch: amd64, dist: jammy, package-type: DEB, repo: microsoft-ubuntu-jammy-prod-apt },
+            { os: ubuntu, version: 22.04, arch: arm64, dist: jammy, package-type: DEB, repo: microsoft-ubuntu-jammy-prod-apt },
           ]
-        arch: [arm, arm64, amd64]
-        exclude:
-          - target: { os: ubuntu, version: 22.04, dist: jammy, package-type: DEB }
-            arch: arm
     with:
       target: ${{ matrix.target.os }}-${{ matrix.target.version }}
-      arch: ${{ matrix.arch }}
-      repo: microsoft-${{ matrix.target.os }}-${{ matrix.target.dist }}-prod-apt
+      arch: ${{ matrix.target.arch }}
+      repo: ${{ matrix.target.repo }}
       dist: ${{ matrix.target.dist }}
+      # ESRP debian based repos (deb) have channels/rings on repos.
+      # ESRP yum based repos (rpm) have no channels
+      channel: ${{ matrix.target.package-type == 'DEB' && 'prod' || '' }}
       package-type: ${{ matrix.target.package-type }}
     secrets: inherit
-
-  release:
```

Comment:
Are we no longer publishing releases to GitHub?

Reply:
If we want to be able to update releases on GitHub that should also be supported by the gh cli, we just need to detect if the release already exists. I remember seeing an example of this somewhere/can find a link if this is what we intend to do?

Reply:
We will, but we will manually create/edit them, its easier that way

Reply:
Unclear here, so not dropping package release to GitHub, just making them manual instead of automatic? Can you please clarify?

Reply:
Yes why not have them automatic/updated (or at least created automatically)? Then edit if needed?

Reply:
Ok, i see we feel strongly about creating a new release through the workflows. I'll re-introduce in another PR as we are currently fixing an existing release and i don't want to complicate the workflow logic for this one-off update.

Reply:
So then leave this portion here in this PR and change in a next one?

Reply:
yes

Reply:
If going to re-add most of this later it is possible to just add an `if: false` to the job to skip it. I think we did something similar for the `container-build` workflow a while ago.

Reply:
Alright, ill re-introduce with `if: false`. All of the artifact names have changed since the workflow refactor though which is why i decided to just remove it as this will all change.

Reply:
Ahh okay I see! 👍